modules/coreboot

```diff
@@ -83,6 +83,7 @@ $(build)/$(coreboot_dir)/.build: \
 $(build)/$(BOARD)/coreboot.rom: $(build)/$(coreboot_dir)/.build
```

Comment:
something for a follow-on: `coreboot.rom` isn't the most useful output name, something like `Heads-<board>-<tag>.rom` would be far better. When building for a half dozen+ boards, having to rename manually to something useful after every build becomes very tiresome.

Reply:
Until now I only added `heads-` to the copy of `coreboot.rom`. If I rename the `coreboot.rom` I need to change the CI files and maybe other things too, right?

Reply:
@alex-nitrokey : CircleCI backups the content of the build dir altogether, while gitlabci is currently foo bar.